GPyOpt/plotting/plots_bo.py

```diff
@@ -122,7 +134,7 @@ def plot_acquisition(bounds, input_dim, model, Xdata, Ydata, acquisition_functio
         plt.subplot(1, 3, 3)
         plt.contourf(X1, X2, acqu_normalized,100)
         plt.colorbar()
-        plt.plot(suggested_sample[:,0],suggested_sample[:,1],'k.', markersize=10)
```

Comment:
btw, this change does not seem necessary. are you a fan of magenta?

Reply:
It's just for visibility against the `viridis` - I found the black as a marker against the cool color background to be tough to spot. But this is, as you point out, not directly relevant to the meat of the PR. For what it's worth, a white, red, hot orange, or pink dot would also work - but I would posit that the differing color between the rightmost and two left plots illustrates that the now-magenta point hasn't been evaluated yet, in contrast to those redish points on the two left plots.

Reply:
ok, convinced, thanks for the explanation